src/App.js

```diff
@@ -4,16 +4,35 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    userRequest: '',
+  };
+
+  searchFilm = (event) => {
+    this.setState({
+      userRequest: event.target.value,
+    });
+  }
 
   render() {
+    const visibleFilms = moviesFromServer.filter(
+      m => m.title.toLowerCase().includes(
+        this.state.userRequest.toLowerCase(),
+      )
+        || m.description.toLowerCase().includes(
+          this.state.userRequest.toLowerCase(),
+        ),
+    );
+
     return (
       <div className="page">
         <div className="page-content">
           <div className="box">
             <div className="field">
               <label htmlFor="search-query" className="label">
                 Search movie
+                {' '}
+                {this.state.filmName}
```

Comment:
```suggestion
                {this.state. userRequest}
```
Looks like you don't have `filmName` in the state. That value called `userRequest`